packages/radio-button/src/RadioButton.js

```diff
@@ -30,10 +29,6 @@ export default class RadioButton extends Component {
      * Called when user moves focus from the field
      */
     onBlur: PropTypes.func,
-    /**
-     * Called when user clicks on the field
-     */
-    onClick: PropTypes.func,
```

Comment:
This is a breaking change. Regardless of what the underlying fix is, we should avoid changing the API.